rllib/execution/multi_gpu_learner_thread.py

```diff
@@ -104,18 +103,14 @@ def __init__(
 
         self.train_batch_size = train_batch_size
 
-        # TODO: (sven) Allow multi-GPU to work for multi-agent as well.
-        self.policy = self.local_worker.policy_map[DEFAULT_POLICY_ID]
+        self.policy_map = self.local_worker.policy_map
```

Comment:
The new changes in this file make it possible to run learner threads (e.g. IMPALA) with multi-agent as well. Before, the somewhat obsoleted "simple_optimizer" would have to be used to make this work.